pyresample/test/test_bucket.py

```diff
@@ -166,7 +173,8 @@ def test_get_average(self):
 
```

Comment:
The last test now fails. Is it obsolete now?

Reply:
Yes, I think `NaN` is a better default result. So maybe change the test on line 172 to `self.assertTrue(np.all(result))`.

Reply:
I replaced it with `self.assertTrue(np.all(np.isnan(result)))` to make it more clear